independent-projects/resteasy-reactive/common/runtime/src/main/java/org/jboss/resteasy/reactive/RestResponse.java

```diff
@@ -0,0 +1,2136 @@
+package org.jboss.resteasy.reactive;
+
+import java.io.InputStream;
+import java.lang.annotation.Annotation;
+import java.net.URI;
+import java.util.Date;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Set;
+import javax.ws.rs.ProcessingException;
+import javax.ws.rs.core.CacheControl;
+import javax.ws.rs.core.EntityTag;
+import javax.ws.rs.core.GenericEntity;
+import javax.ws.rs.core.GenericType;
+import javax.ws.rs.core.Link;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.MultivaluedMap;
+import javax.ws.rs.core.NewCookie;
+import javax.ws.rs.core.NoContentException;
+import javax.ws.rs.core.Response.Status.Family;
+import javax.ws.rs.core.Response.StatusType;
+import javax.ws.rs.core.UriBuilder;
+import javax.ws.rs.core.UriInfo;
+import javax.ws.rs.core.Variant;
+import javax.ws.rs.ext.MessageBodyReader;
+import javax.ws.rs.ext.MessageBodyWriter;
+import javax.ws.rs.ext.RuntimeDelegate;
+
+/**
+ * Defines the contract between a returned instance and the runtime when
+ * an application needs to provide meta-data to the runtime.
+ * <p>
+ * An application class should not extend this class directly. {@code Response} class is
+ * reserved for an extension by API implementation providers. An application should use one
+ * of the static methods to create a {@code Response} instance using a ResponseBuilder.
+ * </p>
+ * <p>
+ * Several methods have parameters of type URI, {@link UriBuilder} provides
+ * convenient methods to create such values as does {@link URI#create(java.lang.String)}.
+ * </p>
+ *
+ * @author Paul Sandoz
```

Comment:
Let's get rid of these

Reply:
I did wonder about that, but it felt like ripping off their author names when most of the API is theirs… So, not sure? OTOH we're changing it, so it could also be misconstrued as blaming them for our changes…

Reply:
My comment was made more from a user perspective since all this stuff will likely be confusing to them

Reply:
OK done.